test/integration/docker/dsd_listening.py

```diff
@@ -5,7 +5,7 @@
 # The DOCKER_IMAGE envvar is needed to specify what
 # image to test
 
-SOCKET_PATH = "/tmp/statsd.socket"
+SOCKET_PATH = b"/tmp/statsd.socket"
```

Comment:
All the output from containers in the `docker` package is returned as bytes ([example](https://docker-py.readthedocs.io/en/stable/containers.html#docker.models.containers.Container.exec_run)). 

For this particular case, the script checks if `SOCKET_PATH` is in the output of some containers so it was less verbose to make it a byte string literal than doing it the other way around.

This was not an issue in Python 2 since the distinction didn't really exist.